.github/renovate.json5

```diff
@@ -6,6 +6,9 @@
   "poetry": {
     "ignorePaths": []
   },
+  "regex": {
+    "ignorePaths": []
+  },
```

Comment:
This will allow for the test charms build deps (rust, uv, poetry etc.) to be updated by renovate.